### PR TITLE
Add VideoPress / Videomaker Onboarding Flow

### PR DESCRIPTION
**Pull request by roundhill**
#### Proposed Changes

* Adds a new signup flow at `/start/videopress` for onboarding a user to a new premium site with Videomaker.

Screen Shot 2022-08-09 at 10 48 38 AM

#### Testing Instructions

* Visit `/start/videopress`, and create a new user and site.
* You should be taken to the checkout page with the premium plan in the cart.
* After payment, you should be redirected to the site editor with Videomaker enabled.

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?

PT: pxWta-1pv-p2

### CODE REVIEW

File: client/lib/signup/step-actions/index.js [hunk 1]
```diff
@@ -508,7 +517,16 @@ export function addAddOnsToCart(
...
-	processItemCart( providedDependencies, newCartItems, callback, reduxStore, slug, null, null );
+	processItemCart(
+		providedDependencies,
+		newCartItems,
+		callback,
+		reduxStore,
+		slug,
+		null,
+		null,
+		null
```
Comment (thedebian) [inline client/lib/signup/step-actions/index.js:528]: Maybe isFreeThemePreselected, themeSligWithRepo and lastKnownFlow can be defaulted to null in processItemCart's prototype ?

**Comment by thedebian** [comment]
Something I noticed while testing, not sure if it should be fixed here or in another PR though;
If the user creation step fails, I can't edit the information again (here I typed my username as my password):
Capture d’écran 2022-08-12 à 09 04 07

**Comment by thedebian** [comment]
Same as previously, I'm not sure it belongs in this PR, but I find it odd the WordPress login appears hidden at the bottom of the page when it should kind of be the primary mean instead of Google or Apple :
Capture d’écran 2022-08-12 à 09 15 25

**Review by thedebian — APPROVED** [review]
It worked ! I stumbled upon some weird things, but I don't think it's related to this PR in particular.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7482380

Thank you @roundhill for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7482380).

**Pull request merged**